tools/topology/sof/pipe-pcm-media.m4

```diff
@@ -58,18 +58,18 @@ W_PCM_PLAYBACK(PCM_ID, Media Playback, 2, 0)
 W_PGA(0, PIPELINE_FORMAT, 2, 2, playback_pga_conf, LIST(`		', "PIPELINE_ID PCM PCM_ID Playback Volume"))
 
 # "SRC 0" has 2 sink and source periods.
-W_SRC(0, PIPELINE_FORMAT, 2, 2, media_src_conf)
+W_SRC(0, PIPELINE_FORMAT, 3, 3, media_src_conf)
```

Comment:
comment above seems inconsistent?

Reply:
I think there was a mistake in topology originally. There's similar for W_PGA() also but I didn't change it yep. It could be 3, 3 since the buffers are anyway allocated such way.

Reply:
yep, please change comment.

Reply:
It's fixed now. I also changed B0 to 2 periods, no need for 3,  and changed for volume component sink to 3 periods to match B1.